docs/quickstart_guide.md

```diff
@@ -0,0 +1,70 @@
+## Quick start guide
+First, set up a **single node** with `stock-only` configuration
+as described in
+[vHive Quickstart](https://github.com/vhive-serverless/vHive/blob/main/docs/quickstart_guide.md).
+Do not start vHive, as it is not needed.
+The relevant commands for a single-node cluster are reproduced here.
+
+```bash
+git clone --branch knative-worker https://github.com/vhive-serverless/airflow.git
+./airflow/scripts/setup_infrastructure.sh
+```
+
+When [setup_infrastructure.sh](../scripts/setup_infrastructure.sh) is finished, run script below.
```

Comment:
It would be nice to have a few sentences explaining what are these scripts doing.